core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -349,7 +349,18 @@
       "Memory to provide to batchwriter to replay mutations for replication"),
   TSERV_ASSIGNMENT_MAXCONCURRENT("tserver.assignment.concurrent.max", "2", PropertyType.COUNT,
       "The number of threads available to load tablets. Recoveries are still performed serially."),
-
+  TSERV_COMPRESSOR_FACTORY("tserver.compressor.factory.class", "org.apache.accumulo.core.file.rfile.bcfile.codec.CompressorFactory", PropertyType.CLASSNAME,
+      "Tablet Server configuration for the compressor factory that will be used when requesting compressors."),
+  TSERV_COMPRESSOR_IN_BUFFER("tserver.compressor.factory.input.buffer.size", "1K", PropertyType.MEMORY,
+      "Tablet Server configuration for the compressor factory that adjusts the input buffer size. Zero uses the full compression block size."),
+  TSERV_COMPRESSOR_OUT_BUFFER("tserver.compressor.factory.output.buffer.size", "1K", PropertyType.MEMORY,
+      "Tablet Server configuration for the compressor factory that adjusts the output buffer size. Default uses the full compression block size."),
+  TSERV_COMPRESSOR_POOL_IDLE("tserver.compressor.pool.max.idle", "25", PropertyType.COUNT,
```

Comment:
Was compressor pooing was unlimited before this change?  If so, is this default too low (I am not saying it is, I have no idea)?  How much time does it take to allocate a decompressor?  I am wondering about a case where I get a burst of scans like 300 and then a few seconds later get another burst, do you have a sense of how that scenario perform before and after this change w/ this default?